### PR TITLE
Aligning S&C tabs with theguardian.com style

### PR DESCRIPTION
**Pull request by jesseyuen**
## Why are you doing this?

The tab component in S&C needed visual alignment with .com.

It's not yet as tidy as it can be and will be revisited before it goes into Storybook but will do for now!

[**Trello Card**](https://trello.com/c/ecK4Jpqq/2146-print-product-page-optimisation)

## Changes

* Style updates to tab component

## Screenshots

screen shot 2019-01-29 at 13 49 41

### CODE REVIEW

File: assets/components/productPage/productPageTabs/productPageTabs.scss [hunk 1]
```diff
@@ -13,24 +13,47 @@
...
   border: 0;
   margin-right: -1px;
-  box-shadow: inset 0 0 0 1px gu-colour(garnett-neutral-4); 
+  box-shadow: inset 0 1px 0 0 gu-colour(garnett-neutral-4);
   color: gu-colour(garnett-neutral-1);
   text-align: left;
   appearance: none;
   box-sizing: border-box;
+  transition: box-shadow 0.2s;
+  font-family: $gu-headline;
```
Comment (walaura) [inline assets/components/productPage/productPageTabs/productPageTabs.scss]: can we turn this into a `@gu-fontset-` mixin?

File: assets/components/productPage/productPageTabs/productPageTabs.scss [hunk 2]
```diff
@@ -13,24 +13,47 @@
...
 }
 
+.component-product-page-tabs__li:last-child {
+  box-shadow: 1px 0 0 0 gu-colour(garnett-neutral-4);
+}
+
+.component-product-page-tabs__li + .component-product-page-tabs__li .component-product-page-tabs__tab {
+  &:before {
+    position: absolute;
+    bottom: 1.0625rem;
```
Comment (walaura) [inline assets/components/productPage/productPageTabs/productPageTabs.scss]: how is this number calculated?

Comment (jesseyuen) [reply assets/components/productPage/productPageTabs/productPageTabs.scss]: Good pick up! Updated.

File: assets/components/productPage/productPageTabs/productPageTabs.scss (hunk 1 above)
Comment (jesseyuen) [reply assets/components/productPage/productPageTabs/productPageTabs.scss]: done!

**Pull request merged**